index.html

```diff
@@ -111,7 +116,7 @@ <h3 class="finaltext"> I can help you build a product , feature or website Look
             <a class="cv" href="">Get My Resume</a>
```

Comment:
Great design @IngDavidTM 🤝

- [ ] As stated in the project requirements, kindly implement the button interactions of `hover` and `enable` on this `Get My Resume` button.